manta-accounting/src/wallet/mod.rs

```diff
@@ -577,6 +578,12 @@ where
 
     /// Ledger Connection Error
     LedgerConnectionError(L::Error),
+
+    /// Missing Spending Key Error
+    MissingSpendingKey,
+
+    /// Missing Proof Authorization Key Error
+    MissingPAKey,
```

Comment:
Just use the full name here.

```suggestion
    MissingProofAuthorizationKey,
```